reference/cd-gitlab-ci.md

```diff
@@ -0,0 +1,183 @@
+---
+title: GitLab CI
+---
+
+This page details how to use [GitLab CI](https://about.gitlab.com/features/gitlab-ci-cd/) to manage deploying
+staging and production stacks based on commits to specific Git branches. This is sometimes
+referred to as Push-to-Deploy.
+
+Pulumi doesn't require any particular arrangement of stacks or workflow to work in a
+continuous integration / continuous deployment system. So the steps described here can be
+altered to fit into any existing type of deployment setup.
+
+## Prerequisites
+- An account on https://app.pulumi.com and that you have created a new project.
+  - As of this writing, we support logins only using GitHub identity.
```

Comment:
I'd drop this line and the next one. We do support non-GitHub forms of identity, they just aren't enabled for everyone in the community edition yet. So it's probably better to just mention "logging in" and leave it at that for the time being.